lib/api/kuzzle.js

```diff
@@ -52,7 +52,7 @@ const
 // Add meaningful stacktrace in Node.js 10
 // @todo must be removed in Node.js 12
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)